extensions/2.0/Vendor/GRIFFEL_bim_data/schema/GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Serializable Object",
+    "type": "object",
+    "description": "Contains same properties as embedded version plus properties to node map. Used only when data is saved in a separate file.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "nodeProperties": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "description": "Node properties and type.",
+                "properties": {
+                    "node": {
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of the node in glTF file."
+                    },
+                    "properties": {
+                        "type": "array",
```

Comment:
Consider adding `"uniqueItems": true` here.

Reply:
Added here and for `propertyNames` and `propertyValues` in _glTF.GRIFFEL_bim_data.schema.json_, for `properties` in _node.GRIFFEL_bim_data.schema.json_.